src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,15 @@
+public class StringAddCalculator {
+
+    public static int splitAndSum(String value) {
+        NumberOperator numberOperator = new NumberOperator();
+        StringOperator stringOperator = new StringOperator(value);
+
+        if (stringOperator.isNotBlank()) {
+            String[] numberArr = stringOperator.split();
+
+            numberOperator.parseIntList(numberArr).forEach(numberOperator::sum);
+        }
```

Comment:
static 메서드를 사용하시면 

```suggestion
            NumberOperator.parseIntList(numberArr).forEach(numberOperator::sum);
        }
```

이 맞습니다.

Reply:
static을 제거하였는데 수정이 덜된 것 같습니다ㅠ static 선언을 제거하여서 수정하겠습니다~

Reply:
@hyunssooo 안녕하세요. 리뷰 주신 부분 반영완료 하였습니다~ 리뷰 감사합니다:)